app/services/trials.js

```diff
@@ -86,6 +86,8 @@ function _processData(trials) {
 
         // Computed values here
         daysAfterCompletion: today.diff(trial.completion_date, 'days'),
+        daysBeforeCompletion: _cleanDate(trial.completion_date)
+          .diff(today, 'days'),
```

Comment:
If it doesn't have a known completion date, it's not displayed anyway, as defined by the specs. So yes, we're covered.